src/main/resources/org/jenkinsci/plugins/oic/OicSecurityRealm/help-sendScopesInTokenRequest_fr.html

```diff
@@ -1,3 +1,4 @@
 <div>
-    Cette option n'est plus supportée, fournir le scope n'est pas autorisée lors de l'obtention d'un code d'accès avec un code d'autorisation
```

Comment:
this is incorrect
the english is 
https://github.com/jenkinsci/oic-auth-plugin/blob/8d45186d385d9a90b524d2c92443f02d9c72e65e/src/main/resources/org/jenkinsci/plugins/oic/OicSecurityRealm/help-sendScopesInTokenRequest.html#L2

which says this does nothing and is not supported.  the French removes this :-/